src/modules/stats/Stats.tsx

```diff
@@ -0,0 +1,262 @@
+import { FC, ReactNode } from 'react';
+import * as React from 'react';
+import { Box, color, Text, Flex } from '@stacks/ui';
+import { Tooltip } from '@components/tooltip';
+import { Card } from '@components/card';
+import { css } from '@emotion/react';
+import { useStxMarketData } from '@common/hooks/use-current-prices';
+import { MICROSTACKS_IN_STACKS, numberToString, usdFormatterNoDecimals } from '@common/utils';
+import { TiArrowSortedUp, TiArrowSortedDown } from 'react-icons/ti';
+import { useQuery } from 'react-query';
+import { useApi } from '@common/api/client';
+import { useAppSelector } from '@common/state/hooks';
+import { selectActiveNetworkUrl } from '@common/state/network-slice';
+import { DEFAULT_LIST_LIMIT } from '@common/constants';
+import { useBlockList } from '@features/blocks-list/useBlockList';
+import { FaBitcoin } from 'react-icons/fa';
+import { Circle } from '@components/circle';
+import { ExplorerSkeletonLoader } from '@components/loaders/skeleton-common';
+import { InfoCircleIcon } from '@components/icons/info-circle';
+
+const statSectionStyle = css`
+  display: grid;
+  border-right: 1px solid var(--colors-border);
+  padding: 24px;
+  height: 131px;
+  &:last-child {
+    border-right: none;
+  }
+`;
+
+const StatSection: FC<{
+  title: ReactNode;
+  bodyMainText: ReactNode;
+  bodySecondaryText: ReactNode;
+  caption: ReactNode;
+}> = ({ title, bodyMainText, bodySecondaryText, caption }) => (
+  <Box css={statSectionStyle}>
+    <Text color={color('text-title')} fontWeight="500" mb={'9px'} style={{ whiteSpace: 'nowrap' }}>
+      {title}
+    </Text>
+    <Flex
+      fontSize={'27px'}
+      mb={'6px'}
+      color={color('text-title')}
+      alignItems={'baseline'}
+      wrap={'nowrap'}
+    >
+      <Text fontSize={'27px'} mr={'6px'} style={{ whiteSpace: 'nowrap' }}>
+        {bodyMainText}{' '}
```

Comment:
Rather than using a space, could you use a `gap` prop in the parent `Flex` element?